packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -225,14 +226,19 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosSdkChainId> implement
   }
 
   async getTxHistory(input: TxHistoryInput): Promise<TxHistoryResponse> {
+    const requestQueue = input.requestQueue ?? new PQueue()
```

Comment:
coalescing to a default `new PQueue()` simplifies the code by essentially defaulting to an infinite concurrency